### PR TITLE
Update to Quarkus 3

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,10 +10,10 @@
     <maven.compiler.release>17</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus-quinoa.version>1.2.7</quarkus-quinoa.version>
+    <quarkus-quinoa.version>2.0.5</quarkus-quinoa.version>
```
Comment (shivam-sharma7) [inline pom.xml:13]:
```suggestion
    <quarkus-quinoa.version>2.0.6</quarkus-quinoa.version>
```

Comment (shivam-sharma7) [reply pom.xml:13]: update quinoa

File: pom.xml [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>2.16.5.Final</quarkus.platform.version>
+    <quarkus.platform.version>3.1.1.Final</quarkus.platform.version>
```
Comment (shivam-sharma7) [inline pom.xml:16]:
```suggestion
    <quarkus.platform.version>3.1.3.Final</quarkus.platform.version>
```

Comment (shivam-sharma7) [reply pom.xml:16]: update quarkus

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

**Comment by kdubois** [comment]
Update to Quarkus 3 already happened in another commit.  Thanks!

**Comment by ia3andy** [comment]
@kdubois I will update it

**Comment by ia3andy** [comment]
Ok this seems to be working fine thanks

**Pull request merged**